### PR TITLE
fix(offline): Ensure sync always completes and other QA

### PR DESCRIPTION
**Pull request by damassi**
### Description

This fixes an issue where the offline sync would timeout from time to time when attempting to download really large collections. Now, there's a timeout of 10s, and anything after that gets passed over, which allows the queue to proceed as expected (and complete). 

There is a catch, however... had to [fork PromisePool](https://github.com/damassi/promise-pool/commit/4fa04cb49f71edd6f4f2cdda822f772bcbdc627b), point package.json at my fork, and create a new dist for a feature that has yet to be released (`withTimeout`), but according to the author [a release is imminent](https://github.com/supercharge/promise-pool/pull/69#issuecomment-1411635006). 

Also:
- Fixes the flashing status text in offline when retrying sync errors. Now it outputs: "Validating sync", which seems less startling to a partner then mentioning "errors"
- Ensures images load on Android. Seems to be some kind of reanimated thing, so just show the images if android, vs smoother fade-in on iOS. We can return to this.

cc @artsy/mobile-platform 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/screens/Artwork/ArtworkContent/ArtworkContent.tsx [hunk 1]
```diff
@@ -159,14 +158,14 @@ export const ArtworkContent = ({ slug }: { slug: string }) => {
   }
 
   return (
-    <Flex flex={1} mt={`${safeAreaInsets.top}px`}>
+    <Flex height="100%">
```
Comment (damassi) [inline src/app/screens/Artwork/ArtworkContent/ArtworkContent.tsx:161]: fixes images not being centered on android in Artwork view. Seems fine for iOS

File: src/app/screens/Settings/OfflineModeSettings.tsx [hunk 2]
```diff
@@ -99,17 +99,21 @@ export const OfflineModeSettings = () => {
...
+                        top: 1,
                       }}
-                    />{" "}
+                    />
+                  </Text>
+                  <Text ml="2px" color="onPrimaryHigh">
                     {syncProgress}
                   </Text>
-                </>
+                </Flex>
```
Comment (damassi) [inline src/app/screens/Settings/OfflineModeSettings.tsx:116]: Fix ... alignment in android offline

File: src/app/system/wrappers/CachedImage.tsx [hunk 3]
```diff
@@ -23,7 +23,10 @@ export const CachedImage: React.FC<CachedImageProps> = ({
   ...restProps
 }) => {
   const isDoneLoading = useRef(false)
-  const opacity = useSharedValue(fadeInOnLoad ? 0 : 1)
+
+  const initialOpacity = Platform.OS === "ios" ? 0 : 1
+  const opacity = useSharedValue(fadeInOnLoad ? initialOpacity : 1)
```
Comment (damassi) [inline src/app/system/wrappers/CachedImage.tsx:28]: Skip fade on android (for now)

File: src/app/system/sync/syncManager.ts [hunk 4]
```diff
@@ -368,7 +377,8 @@ export function initSyncManager({
 
     await PromisePool.for(urls)
       .onTaskStarted(reportProgress("Syncing images"))
-      .withConcurrency(50)
+      .withConcurrency(20)
```
Comment (pvinis) [inline src/app/system/sync/syncManager.ts:380]: fun api

**Review by pvinis — APPROVED** [review]

File: src/app/screens/Artwork/ArtworkContent/ArtworkContent.tsx [hunk 5]
```diff
@@ -159,14 +158,14 @@ export const ArtworkContent = ({ slug }: { slug: string }) => {
...
       <ImageModal
         isModalVisible={isModalVisible}
         setIsModalVisible={setIsModalVisible}
         uri={image?.resized?.url ?? ""}
       />
 
-      <Flex height={imageFlexHeight} px={2}>
+      <Flex px={`${space(2)}px`}>
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/app/screens/Artwork/ArtworkContent/ArtworkContent.tsx:168]: oh duh lol

**Pull request merged**